File: 프로그래머스/0/120821. 배열 뒤집기/배열 뒤집기.cs
using System;
using System.Collections.Generic;

public class Solution 
{
    public int[] solution(int[] num_list) 
    {
        int nl = num_list.Length;
        List<int> reverseList = new List<int>(num_list);
        reverseList.Reverse();
        
        return reverseList.ToArray();
        
    }
}

File: 프로그래머스/0/120821. 배열 뒤집기/배열 뒤집기.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> num_list) 
{
    vector<int> rnum_list;
    for(int i=num_list.size()-1; i>=0; i--)
    {
        rnum_list.push_back(num_list[i]);
    }
    return rnum_list;
}